Application/registerationForm.js

```diff
@@ -0,0 +1,49 @@
+document.getElementById('registerationForm').addEventListener('submit', function (event) {
+  event.preventDefault();
+
+  if (emailCheck() && contactCheck()) {
+    alert("Registeration done sucessfull");
+    location.replace("home.html");
+  } else {
+    alert("Registeration not sucessfull");
+
+  }
+});
+
+function passwordShow() {
+  const password = document.getElementById("Password");
+
+  if (password.type === "password") {
+    password.type = "text";
+  } else {
+    password.type = "password";
+  }
+}
+
+function emailCheck() {
```

Comment:
A more appropriate name for the `emailCheck` function could be validateEmail or `isValidEmail`.
https://www.linkedin.com/pulse/art-writing-clean-functions-code-practices-mominur-rahman-jurqc#:~:text=When%20naming%20functions%2C%20be%20clear,its%20role%20within%20the%20code.